code/modules/networks/computer3/sec_rec.dm

```diff
@@ -587,6 +587,9 @@
 				var/list/datum/db_record/results = list()
 				for(var/datum/db_record/R as anything in data_core.general.records)
 					var/haystack = jointext(list(ckey(R["name"]), ckey(R["dna"]), ckey(R["id"]), ckey(R["fingerprint"]), ckey(R["rank"])), " ")
+					var/haystack_secure_addition = data_core.security.find_record("name", R["name"])
```

Comment:
why name and not ID?

Reply:
Not sure! ID probably would've been better! It's more or less my first dive into it at all, so a bit of a learning experience. I'll change it to use ID instead, shortly.